src/igcontrolbase/igcontrolbase.ts

```diff
@@ -230,6 +230,9 @@ export class IgControlBase<Model> implements DoCheck {
 		if (o1 === o2) { return true; }
 		if (o1 === null || o2 === null) { return false; }
 		if (o1 !== o1 && o2 !== o2) { return true; }// NaN === NaN
+		if (o1.nodeName) {
```

Comment:
@tbelcheva The fix is ok, however there might be a case where the customer object has "nodeName" property and in this case the equalsDiff will not work. Please add some more conditions in order to make sure that this is indeed a DOM node. Here is some suggestion: https://stackoverflow.com/questions/384286/javascript-isdom-how-do-you-check-if-a-javascript-object-is-a-dom-object